docs/UserGuide.md

```diff
@@ -175,87 +181,99 @@ You can find all your activities on a specific date in AtheltiCLI.
 
 * `find-activity 2021-09-01`
 
+---
+
 ### 🎯 Setting Activity Goals:
 
 `set-activity-goal`
 
-You can set goals for your activities in AthletiCLI by setting the target distance or duration for a specific sport.
-The goals can track your daily, weekly, monthly, or yearly progress.
+You can set goals for specific sports by defining target distance or duration over various periods.
```

Comment:
```suggestion
You can set goals for specific sports by defining the target distance or duration over various periods. The goals can track your daily, weekly, monthly, or yearly progress.
```
I think it's good to mention the periods here so that the user better understands the feature.